src/components/DynamicSingleStat.res

```diff
@@ -282,9 +289,13 @@ let make = (
           source,
           prefix: ?urlConfig.prefix,
         }
-        let singleStatBodyMakerFn = urlConfig.singleStatBody->Option.getOr(singleStatBodyMake)
 
-        let singleStatBody = singleStatBodyMakerFn(singleStatBodyEntity)
+        let singleStatBodyMakerFn = switch formaPayload {
```

Comment:
api is getting called twice if we apply some filters

https://github.com/juspay/hyperswitch-control-center/assets/120017870/1ebaab4a-4355-4822-87bd-f81c4b0e2982
<img width="1728" alt="image" src="https://github.com/juspay/hyperswitch-control-center/assets/120017870/ac2af780-1226-4497-a5dd-bdd631b85a16">
<img width="1728" alt="image" src="https://github.com/juspay/hyperswitch-control-center/assets/120017870/9698ba0a-ec1f-482c-8891-4d49382bb5d5">
Try moving this to separate function and call that function from useEffect

Reply:
single state has a two api calls, one returns the single value and another returns a series value so when filters are changes two api calls happens, delta value is diff btw both the api calls